feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableShimmerListItem.kt

```diff
@@ -67,14 +87,23 @@ fun TimetableShimmerListItem(modifier: Modifier = Modifier) {
     }
 }
 
-// TODO: Need to resolve CI unit test failures due to TimetableShimmerListItemPreview()
-// @MultiThemePreviews
-// @MultiLanguagePreviews
-// @Composable
-// fun TimetableShimmerListItemPreview() {
-//    KaigiTheme {
-//        Surface {
-//            TimetableShimmerListItem()
-//        }
-//    }
-// }
+private fun isRobolectric(): Boolean {
+    @Suppress("SwallowedException")
+    return try {
+        Class.forName("org.robolectric.Robolectric")
+        true
+    } catch (e: ClassNotFoundException) {
+        false
+    }
+}
```

Comment:
To be honest, this isn't an ideal way to determine it.
I would appreciate it if Robolectric itself provided an API to determine if it's currently being executed... 😔

Reply:
I really use this implementation(Class.forName for checking tests). But could you check `isTest()` method?

Reply:
Oops! I did not know that function existed. Thanks!